src/einsteinpy/symbolic/riemann.py

```diff
@@ -55,7 +55,10 @@ def __init__(
     @classmethod
     def from_christoffels(cls, chris, parent_metric=None):
         """
-        Get Riemann Tensor calculated from a Christoffel Symbols
+        Get Riemann Tensor calculated from a Christoffel Symbols Reimann Tensor is given as:
```

Comment:
Please remove the 'a' before "Christoffel" and add a period / full-stop, after "Symbols". Also, there's a misspelling. "Reimann" should be "Riemann", so that it reads:

> Get Riemann Tensor calculated from Christoffel Symbols. Riemann Tensor is given as:

Reply:
Let's just say this:

> Get Riemann Tensor from Christoffel Symbols. Riemann Tensor is given as: